src/telemetry/metrics/mod.rs

```diff
@@ -84,9 +68,9 @@ pub fn new(
     process: process::Report,
     transport_report: transport::Report,
     tls: tls_config_reload::Report
-) -> (Record, Serve) {
+) -> (http::Record, Serve) {
     let metrics = Arc::new(Mutex::new(Root::new(process, transport_report, tls)));
-    (Record::new(&metrics), Serve::new(&metrics, idle_retain))
+    (http::Record::new(&metrics), Serve::new(&metrics, idle_retain))
```

Comment:
I'm guessing this will eventually be changed to follow the pattern used by the other metrics record/report types?

Reply:
yeah, though it may take a few more PRs to get there